### PR TITLE
Fault page

### PR DESCRIPTION
**Pull request by augustt198**

### CODE REVIEW

**Review by nistath — CHANGES_REQUESTED** [review]
Recoverable contactor error should be called TSMS. Very good if tested otherwise.

**Comment by augustt198** [comment]
Tested it's able to print multiple faults. Didn't test with enough faults to see if it handles wrapping correctly.

File: src/dashboard/src/page_manager.c [hunk 1]
```diff
@@ -116,40 +116,16 @@ void draw_critical_page(page_manager_t *pm, NHD_US2066_OLED *oled) {
...
-            oled_print(oled, "RECOV");
-        }
-        else if (pm->stats->error_state == can0_VCUHeartbeat_error_state_FATAL_ERROR_STATE) {
-            oled_print(oled, "FATAL");
-        }
+
+    if (pm->stats->error_state == can0_VCUHeartbeat_error_state_RECOVERABLE_ERROR_STATE) {
+        oled_print(oled, "R");
+    } else if (pm->stats->error_state == can0_VCUHeartbeat_error_state_FATAL_ERROR_STATE) {
+        oled_print(oled, "F");
```
Comment (nistath) [inline src/dashboard/src/page_manager.c:123]: UX Note: Add an indication of whether an ESTOP or TSMS is open.

Comment (nistath) [reply src/dashboard/src/page_manager.c:123]: Also maybe expand to RECOV and FATAL since we have the space.

**Review by nistath — APPROVED** [review]
Thanks! Looks good, but don't merge until tested please!

**Comment by nistath** [comment]
@augustt198, for future reference, you should be able to boot the car up with an ESTOP hit and the BMS CAN disconnected (5 faults). Or just manually edit your struct such that there are fake faults.

**Pull request merged**